app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,13 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.1/font/bootstrap-icons.css">
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= vite_client_tag %>
     <%= vite_javascript_tag 'application' %>
+
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
I surprised you need this. we have jquery included here:https://github.com/pulibrary/tiger-data-app/blob/724245ed452bf2e80e113d573fa8ca055ee2cd00/vite.config.ts#L8

Reply:
I wonder if it is because we kept the JavaScript on the page instead of passing it through the asset pipeline. I've added a ticket to move the JavaScript out (https://github.com/pulibrary/tiger-data-app/issues/296). 

I'll make a note to check if we can remove the extra jQuery line once we implement that ticket.

Reply:
Fixed with @carolyncole PR